### PR TITLE
abseil: update to 20240116.0

### PR DESCRIPTION
**Pull request by herbygillot**
###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Review by judaew — APPROVED** [review]
LGTM, thank you for the contribution ✨

**Review by barracuda156 — APPROVED** [review]
`abseil` 20240116.0 builds fine on PowerPC for me.

With regard to `s2geometry` I rely on CI.

File: devel/abseil/Portfile [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.1
-PortGroup           github 1.0
-PortGroup           legacysupport 1.1
+PortGroup           cmake                       1.1
+PortGroup           github                      1.0
+PortGroup           legacysupport               1.1
```
Comment (mascguy) [inline devel/abseil/Portfile:6]: Do we really need to mess with this?

Some maintainers would strongly prefer that we leave their spacing alone. Please and Thanks.  😉

**Review by mascguy — CHANGES_REQUESTED** [review]

File: devel/abseil/Portfile (hunk 1 above)
Comment (herbygillot) [reply devel/abseil/Portfile]: Acknowledged and reverted.

**Review by mascguy — APPROVED** [review]

**Comment by pmetzger** [comment]
@neverpanic You okay with merging?

**Review by neverpanic — APPROVED** [review]
Yes. In fact the revbump of Bear should no longer required due to the changes we made in re2 IIRC, but it also doesn't hurt.

**Pull request merged**